script/encrypt

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -ueo pipefail
+
+ROOT=$(cd $(dirname $0)/.. && pwd)
+
+source ${ROOT}/script/include/credentials.sh
+
+INSTANCE=$(credential instance)
+SRC=${ROOT}/credentials.yml
+DEST=${1:-${ROOT}}
+
+[ -d "${DEST}" ] && {
+  DEST="${DEST}/credentials-${INSTANCE}.yml.enc"
+}
+
+exec openssl enc -md sha1 -salt -aes-256-cbc -in ${SRC} -out ${DEST}
```

Comment:
@ktbartholomew Care to double-check my incantation here?

I didn't even know openssl could do this until you mentioned it yesterday :wink: